packages/next/src/server/lib/router-utils/filesystem.ts

```diff
@@ -127,6 +127,9 @@ export async function setupFsCheck(opts: {
   const nextStaticFolderItems = new Set<string>()
   const legacyStaticFolderItems = new Set<string>()
 
+  // TODO: (wyattjoh) maybe this isn't updated fast enough, we may need the directory scan from the dev matchers
+  const appRoutes = new AppRouteDefinitionBuilder()
```

Comment:
If this isn't updated frequently enough via WatchPack, we may need to swap this out for the development matcher system.